server/src/main/java/org/elasticsearch/index/mapper/AbstractGeometryFieldMapper.java

```diff
@@ -136,6 +139,14 @@ protected Object parseSourceValue(Object value) {
                 }
             };
         }
+
+        @Override
+        public BlockLoader blockLoader(BlockLoaderContext blContext) {
+            // Currently we can only load from source in ESQL
+            return new BlockSourceReader.GeometriesBlockLoader(
+                valueFetcher(blContext.sourcePaths(name()), nullValue, GeometryFormatterFactory.WKT)
```

Comment:
In https://github.com/elastic/elasticsearch/pull/103461, we added support to generate WKB. I think we should use that here so we don't need to transform the WKT into WKB later, it is wasteful.

Reply:
Good point. I see the option to generate WKB directly in the value fetcher, so we don't go via WKT, but directly to WKB. I'm busy working on GeoPointFieldMapperTests at the moment, so I'll investigate this too.

Reply:
Done